drivers/sensor/bme280/bme280.c

```diff
@@ -245,6 +245,10 @@ static int bme280_channel_get(const struct device *dev,
 			(((data->comp_press & 0xff) * 1000U) >> 8);
 		break;
 	case SENSOR_CHAN_HUMIDITY:
+		/* The BMP280 doesn't have a humidity sensor */
+		if (data->chip_id != BME280_CHIP_ID) {
+			return -EINVAL;
```

Comment:
I would rather use `ENOTSUP/ENODATA`.

Reply:
It will return `EINVAL` for all other unsupported sensor channels, not sure this one should be different.
Let's see what other say. I'll have to remember to change the commit message too! :)

Reply:
The thing is whether the `EINVAL` should be returned for unsupported channels. @MaureenHelm what are your thoughts here?

Reply:
`ENOTSUP` should be returned for unsupported channels